ocs_ci/ocs/utils.py

```diff
@@ -650,6 +655,13 @@ def collect_ocs_logs(dir_name):
     log.info(f"OCS logs will be placed in location {dump_dir}")
     occli = OCP()
     try:
-        occli.exec_oc_cmd(cmd, out_yaml_format=False)
+        occli.exec_oc_cmd(
+            cmd, out_yaml_format=False, timeout=must_gather_timeout
+        )
     except CommandFailed as ex:
         log.error(f"Failed during must gather logs! Error: {ex}")
+    except TimeoutExpired as ex:
+        log.error(
+            f"Timeout {must_gather_timeout}s for must-gather reached, command"
+            f" exited with error: {ex}"
```

Comment:
I think you need {str(ex)} instead of {ex}

Reply:
no need at all Elad, see output from ipython, the python do the conversion itself and calling __str__:
```
In [1]: try:
   ...:     raise Exception("This is my exception for Elad :)")
   ...: except Exception as ex:
   ...:     print(f"Exception raised: {ex}")
   ...:
Exception raised: This is my exception for Elad :)
```

Here you have another examples which are redundant:
```

In [2]: try:
   ...:     raise Exception("This is my exception for Elad :)")
   ...: except Exception as ex:
   ...:     print(f"Exception raised: {ex.__str__()}")
   ...:
   ...:
Exception raised: This is my exception for Elad :)

In [3]: try:
   ...:     raise Exception("This is my exception for Elad :)")
   ...: except Exception as ex:
   ...:     print(f"Exception raised: {str(ex)}")
   ...:
   ...:
   ...:
Exception raised: This is my exception for Elad :)
```

Reply:
@petr-balogh wasn't sure about this. Thanks for checking